### ISSUE TITLE
Agenda 2021-09-02, 15:30 UTC

### ISSUE THREAD

**Issue by jackgerrits** [VowpalWabbit/agenda#10]
_Please comment on this issue to raise an item to discuss. How to join the meeting can be found [here](https://vowpalwabbit.org/blog/open-meeting.html)._

**Comment by georgefei** [comment]
- Model performance and optimal hyperparameter choice are dependent on the order of the input data to the cb model (more details in https://gitter.im/VowpalWabbit/community?at=610054a39e84ba381e4267bf)

  - Training command: vw --cb 3 -c -d  train_data.txt/part-00000 --passes 10 --decay_learning_rate 0 -l 0.5 --power_t 0 -f vw.model --l1 1e-07
  - Testing command: vw -t -d  test_data.txt/part-00000 -i vw.model -p test_pred.txt

- Using bfgs yields worse than random average loss;  example command vw --cb 3 -c -d train_data.txt -l 0.5 power_t = 0 --bfgs --mem 15 --passes 10 -f vw.model --l2 10.0
